### PR TITLE
chore: add attribute-type

### PR DESCRIPTION
**Pull request by tylermorrisford**
Adds `attribute-type` model.

### CODE REVIEW

File: models/attribute-type/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  },
+  "main": "dist/commercetools-test-data-attribute-type.cjs.js",
+  "module": "dist/commercetools-test-data-attribute-type.esm.js",
+  "files": ["dist", "package.json", "LICENSE", "README.md"],
+  "dependencies": {
+    "@babel/runtime": "^7.17.9",
+    "@babel/runtime-corejs3": "^7.17.9",
+    "@commercetools-test-data/commons": "4.4.0",
+    "@commercetools-test-data/core": "4.4.0",
+    "@commercetools-test-data/tax-rate": "4.4.0",
```
Comment (jaikamat) [inline models/attribute-type/package.json:24]: Is TaxRate used in this package?

Comment (tylermorrisford) [reply models/attribute-type/package.json:24]: To quote one of my favorite people, "rogue copy pasta strikes again"

File: models/attribute-type/src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  money: 'MoneyAttributeDefinitionType',
+  nested: 'NestedAttributeDefinitionType',
+  number: 'NumberAttributeDefinitionType',
+  reference: 'ReferenceAttributeDefinitionType',
+  set: 'SetAttributeDefinitionType',
+  text: 'TextAttributeDefinitionType',
+  time: 'TimeAttributeDefinitionType',
+};
+
+export const typeNames = Object.keys(graphqlTypenameByAttributeTypeName);
```
Comment (jaikamat) [inline models/attribute-type/src/constants.ts:17]: I think this might be unused 🤔

File: models/attribute-type/src/types.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import type { AttributeType } from '@commercetools/platform-sdk';
+import type { TBuilder } from '@commercetools-test-data/core';
+
+export type TAttributeType = AttributeType;
+
+export type TAttributeTypeGraphql = TAttributeType & {
+  __typename: TAttributeType['name'];
```
Comment (jaikamat) [inline models/attribute-type/src/types.ts:7]: ✨

File: models/attribute-type/package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@commercetools-test-data/attribute-type",
+  "version": "4.4.0",
+  "description": "Data model for commercetools API Attribute Type",
```
Comment (tylermorrisford) [inline models/attribute-type/package.json:4]: confused here: should I set this as version 4.5 to match the existing?

**Review by stephsprinkle — APPROVED** [review]

File: .changeset/cuddly-squids-allow.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-test-data/attribute-type': minor
+---
+
+Add `attribute-type`
```
[the remark this answers is not in the document]
Comment (tylermorrisford) [reply .changeset/cuddly-squids-allow.md]: 💪

**Pull request merged**